src/Tizen.Applications.UI/Tizen.Applications.CoreBackend/UICoreBackend.cs

```diff
@@ -161,8 +174,7 @@ private void OnAppControlNative(IntPtr appControlHandle, IntPtr data)
             {
                 // Create a SafeAppControlHandle but the ownsHandle is false,
                 // because the appControlHandle will be closed by native appfw after this method automatically.
-                SafeAppControlHandle safeHandle = new SafeAppControlHandle(appControlHandle, false);
-
+                using SafeAppControlHandle safeHandle = new SafeAppControlHandle(appControlHandle, false);
```

Comment:
I'm curious about the intention of putting the `using` keyword.

Reply:
It's for calling Dispose() automatically.